etc/ATSCam/primary_hdu.header

```diff
@@ -7,9 +7,12 @@ ORIGIN  = 'LSST DM Header Service'/ FITS file originator
          ---- Date, night and basic image information ----
 DATE    =                      / Creation Date and Time of File
 DATE-OBS=                      / Date of the observation (image acquisition)
+DATE-BEG=                      / Time at the start of integration
 DATE-END=                      / end date of the observation
 MJD     =                      / Modified Julian Date that the file was written
 MJD-OBS =                      / Modified Julian Date of observation
+MJD-BEG =                      / Modified Julian Day derived from DATE-BEG
```

Comment:
I'd say "corresponding to" rather than "derived from" but I'm not hung up on that. More importantly it's "Date" not "day" (day implies integer).

Reply:
Changed Day --> Date